cs/Openfin.PlatformAPI.Demo/Openfin.PlatformAPI.Demo/layout.json

```diff
@@ -0,0 +1,68 @@
+{  
+        "settings": {
+            "hasHeaders": true,
+            "constrainDragToContainer": true,
+            "reorderEnabled": true,
+            "selectionEnabled": false,
+            "popoutWholeStack": false,
+            "blockedPopoutsThrowError": true,
+            "closePopoutsOnUnload": true,
+            "showPopoutIcon": false,
+            "showMaximiseIcon": false,
+            "showCloseIcon": false,
+            "responsiveMode": "onload",
+            "tabOverlapAllowance": 0,
+            "reorderOnTabMenuClick": true,
+            "tabControlOffset": 10
+        },
+        "dimensions": {
+            "borderWidth": 5,
+            "borderGrabWidth": 15,
+            "minItemHeight": 10,
+            "minItemWidth": 10,
+            "headerHeight": 20,
+            "dragProxyWidth": 300,
+            "dragProxyHeight": 200
+        },
+        "labels": {
+            "close": "close",
+            "maximise": "maximise",
+            "minimise": "minimise",
+            "popout": "open in new window",
+            "popin": "pop in",
+            "tabDropdown": "additional tabs"
+        },
+        "content": [
+            {
+                "type": "stack",
+                "id": "no-drop-target",
+                "isClosable": true,
+                "reorderEnabled": true,
+                "title": "",
+                "activeItemIndex": 0,
+                "content": [
+                    {
+                        "type": "component",
+                        "componentName": "view",
+                        "componentState": {
+                            "name": "%name%",
```

Comment:
let's make sure to remove the name here. (I guess it will depend on adapter changes as name is required.)